Python/generated_cases.c.h

```diff
@@ -3547,18 +3547,18 @@
 
         TARGET(CALL_FUNCTION_EX) {
             PREDICTED(CALL_FUNCTION_EX);
-            PyObject *func, *callargs, *kwargs = NULL, *result;
-            if (oparg & 0x01) {
-                kwargs = POP();
+            PyObject *kwargs = (oparg & 1) ? PEEK(((oparg & 1) ? 1 : 0)) : NULL;
+            PyObject *callargs = PEEK(1 + ((oparg & 1) ? 1 : 0));
+            PyObject *func = PEEK(2 + ((oparg & 1) ? 1 : 0));
+            PyObject *null = PEEK(3 + ((oparg & 1) ? 1 : 0));
+            PyObject *result;
```

Comment:
There’s an unused variable warning on null.

Reply:
Oh what a pain. I'll have to uglify the code to handle this.